ansible_roles/managed_host/tasks/ssh_hardening.yml

```diff
@@ -142,12 +148,12 @@
   register: disable_rsa_host_key_result
   shell: "find /etc/ssh -iname '*_rsa_*' -not -name '*INSECURE' -exec rename -v 's/(.*)/$1.INSECURE/' {} \\;"
   changed_when: disable_rsa_host_key_result.stdout != ""
-  when: check_rsa_host_key_length.rc == 0
+  when: check_rsa_host_key_length.rc == 1
```

Comment:
This is a bit confusing, in shell exit code `0` means success (i.e. _true_). I will revert those two lines.